packages/controller-utils/src/constants.ts

```diff
@@ -84,3 +84,29 @@ export const OPENSEA_PROXY_URL =
   'https://proxy.metafi.codefi.network/opensea/v1/api/v1';
 export const OPENSEA_API_URL = 'https://api.opensea.io/api/v1';
 export const OPENSEA_TEST_API_URL = 'https://testnets-api.opensea.io/api/v1';
+
+/**
+ * Approval request types for various operations.
+ * These types are used by different controllers to create and manage
+ * approval requests consistently.
+ */
+export enum ApprovalType {
+  AddEthereumChain = 'wallet_addEthereumChain',
+  EthAccounts = 'eth_accounts',
+  EthDecrypt = 'eth_decrypt',
+  EthGetEncryptionPublicKey = 'eth_getEncryptionPublicKey',
+  EthRequestAccounts = 'eth_requestAccounts',
+  EthSign = 'eth_sign',
+  EthSignTypedData = 'eth_signTypedData',
+  EthSignTypedDataV3 = 'eth_signTypedData_v3',
```

Comment:
We use the same approval type for all versions so shouldn't need the V3 and V4 values.